generators/client/files-angular.js

```diff
@@ -305,6 +305,7 @@ const files = {
                 'admin/user-management/_user-management-detail.component.ts',
                 'admin/user-management/_user-management-dialog.component.ts',
                 'admin/user-management/_user-management-delete-dialog.component.ts',
+                'admin/user-management/_user-modal.service.ts',
                 'admin/user-management/_user.model.ts',
                 'admin/user-management/_user.service.ts',
                 { file: 'admin/user-management/_user-management.state.ts', method: 'copyJs' },
```

Comment:
state files no longer needs to be passed through `copyJs` as there is no i18n stuff to strip anymore so this can just be a single line file name entry. Same for every other state/route files